reco_utils/recommender/newsrec/models/base_model.py

```diff
@@ -0,0 +1,289 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from os.path import join
+import abc
+import time
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+from reco_utils.recommender.deeprec.deeprec_utils import cal_metric
+
+
+__all__ = ["BaseModel"]
+
+
+class BaseModel:
```

Comment:
better to use new style class here since we're still support python 2
```suggestion
class BaseModel(object):
```

Reply:
afaik we are not supporting python 2:
- https://github.com/microsoft/recommenders/blob/master/scripts/generate_conda_file.py#L38
- https://github.com/microsoft/recommenders/blob/master/setup.py#L42
is there any place where we have python 2 stuff that I missed?

Reply:
Ah. That's great. Nevermind then.